docs/advanced/document-store/index.md

```diff
@@ -346,6 +346,179 @@ array([[7., 7., 7.],
        [4., 4., 4.]])
 ```
 
+## Persistence, mutations and context manager
+
+Having DocumentArrays that are backed by an external store does however introduce an extra consideration into the way one thinks about DocumentArrays.
```

Comment:
```suggestion
Having DocumentArrays that are backed by a document store introduces an extra consideration into the way you think about DocumentArrays.
```

Small thing, but let's not assume that people have read the section above. They might have come from an outside link, or just scanned over the rest bc they very looking specifically for this section.

Also, we generally prefer `you` over `one` or `we`.